doc/api/process.md

```diff
@@ -1593,6 +1593,26 @@ Will generate:
 When using [`Worker`][] threads, `rss` will be a value that is valid for the
 entire process, while the other fields will only refer to the current thread.
 
+The `process.memoryUsage()` method iterate over each page to gather
+informations about memory usage which can be slow depending on the
```

Comment:
I find `can be slow` quite vague - in special as a caller can't do anything to avoid this to test if it will be slow.
Are we talking about µs, ms, or seconds here?

Reply:
It depends on the program size in memory. Access the RSS take a fixed time but compute heap statistics is quite costly since you have to iterate over every memory page.  
The difference can be in seconds. (see the benchmark in the PR description)

Reply:
The sample in PR description prints milli seconds not seconds so not that bad.
I tried it local on my machine and even for 4GB heaps it was below 0.2ms. But for sure the rss only API is much faster.

Reply:
At the beginning I have done this PR because I found very strange results while benchmarking promises (https://github.com/nodejs/node/issues/33384).  
So even if the difference is not that huge it can be misleading in some cases